dataikuapi/govern/artifact_search_handler.py

```diff
@@ -0,0 +1,203 @@
+import abc
+from abc import ABC
```

Comment:
I think abc is too tied to python version (3.X+) maybe consider not having it for low python versions?